kore/test/Test/Kore/Step/Simplification/Ceil.hs

```diff
@@ -473,49 +478,51 @@ appliedMockEvaluator result =
 
 mockEvaluator
     :: MonadSimplify simplifier
-    => AttemptedAxiom variable
-    -> TermLike variable
-    -> SideCondition variable
-    -> simplifier (AttemptedAxiom variable)
+    => AttemptedAxiom RewritingVariableName
+    -> TermLike RewritingVariableName
+    -> SideCondition RewritingVariableName
+    -> simplifier (AttemptedAxiom RewritingVariableName)
 mockEvaluator evaluation _ _ = return evaluation
 
 mapVariables
     :: forall variable
     .  InternalVariable variable
-    => TestPattern
+    => Pattern RewritingVariableName
     -> Pattern variable
 mapVariables =
     Pattern.mapVariables (pure worker)
   where
-    worker :: VariableName -> variable
-    worker v = fromVariableName v { counter = Just (Sup.Element 1) }
+    worker :: RewritingVariableName -> variable
+    worker v = fromVariableName $ (from v) { counter = Just (Sup.Element 1) }
```

Comment:
Could you add type annotations for the `from` here so that it's more obvious into what we're transforming the `v`?